composer/core/time.py

```diff
@@ -44,6 +44,7 @@ class TimeUnit(StringEnum):
     SAMPLE = 'sp'
     TOKEN = 'tok'
     DURATION = 'dur'
+    SECOND = 'sec'
```

Comment:
Is there some documentation that should be updated with this new interval specification? prolly [here](https://docs.mosaicml.com/projects/composer/en/latest/trainer/time.html)